<launch>

    <rosparam command="load" file="$(find araig_calculators)/config/test_calculators.yaml" />

    <node pkg="araig_calculators" type="folder_bagger_node" name="test_bagger" output="screen" >
        <remap from="/start" to="/signal/runner/start_robot"/>
        <remap from="/stop" to="/signal/runner/test_completed"/>
        <remap from="/test_failed" to="/signal/runner/test_failed"/>
        <remap from="/test_succeeded" to="/signal/runner/test_succeeded"/>
    </node>

</launch>
